code/controlconfig/controlsconfig.cpp

```diff
@@ -393,22 +393,7 @@ int control_config_detect_axis()
 
 void control_config_conflict_check()
 {
-	int i, j, shift = -1, alt = -1;
-
-	for (i=0; i<CCFG_MAX; i++) {
-		Conflicts[i].key = Conflicts[i].joy = -1;
-		switch (Control_config[i].key_id) {
-			case KEY_LSHIFT:
-			case KEY_RSHIFT:
-				shift = i;
-				break;
-
-			case KEY_LALT:
-			case KEY_RALT:
-				alt = i;
-				break;
-		}
-	}
+	int i, j;
```

Comment:
Damn, I didn't notice that `key` and `joy` get initialized in the loop. I'll add it back in.